### PR TITLE
anytype: 0.35.4 -> 0.35.25

### PR DESCRIPTION
**Pull request by Biepa**
## Description of changes

Updating to the most current version.

First PR request for nixpks. Let me know of issues.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by iblech** [comment]
Welcome to our small community :-) I'm not familiar with Anytype, but I just tested running the executable and this worked. I would merge this PR if I had the permissions. :-)

**Review by iblech — APPROVED** [review]

**Comment by Biepa** [comment]
Thank you for taking the time to check. @iblech 

(Einen schönen Sonntag wünsche ich :smile:)

File: pkgs/applications/misc/anytype/default.nix [hunk 1]
```diff
@@ -2,13 +2,13 @@
 
 let
   pname = "anytype";
-  version = "0.35.4";
+  version = "0.35.25-beta";
```
Comment (Mic92) [inline pkgs/applications/misc/anytype/default.nix:5]: I can merge. Just want to know why you are updating this to an unreleased version instead to an release. Are there important fixes not present in any release?

Comment (Biepa) [reply pkgs/applications/misc/anytype/default.nix:5]: Updating as this is their latest version (I'm a community mod and in contact with the team). It's stable and 0.35.4 is missing quite a few features and fixes. If nothing would speak against it, I would try to keep the master branch (unstable) up to date with their latest release (including beta channel). And for the 23.05 branch I would update it only to their non beta versions.
Let me know of any questions or if I got something wrong with the unstable/stable channel.

**Pull request merged**